sound/soc/soc-dai.c

```diff
@@ -391,6 +391,44 @@ bool snd_soc_dai_stream_valid(struct snd_soc_dai *dai, int dir)
 	return stream->channels_min;
 }
 
+/*
+ * snd_soc_dai_link_set_capabilities() - set dai_link properties based on its DAIs
+ */
+void snd_soc_dai_link_set_capabilities(struct snd_soc_dai_link *dai_link)
+{
+	struct snd_soc_dai_link_component *cpu;
+	struct snd_soc_dai_link_component *codec;
+	struct snd_soc_dai *dai;
+	bool supported[2];
```

Comment:
it gives me a slightly uncomfortable feeling in the stomach - using different symbols in the array and the loop... We probably should use `SNDRV_PCM_STREAM_LAST` for such arrays. But I know 2 is used in many places...

Reply:
I'll do the change.